### PR TITLE
LLM experiments yaml config

### PR DESCRIPTION
**Pull request by tomsilver**
closes #1087

full results: https://docs.google.com/spreadsheets/d/11jM4G_jkk6owaieerXif3cakK57Tu9BLyLdLAeMFyKY/edit#gid=0

### CODE REVIEW

File: scripts/configs/llm_pddl.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+ARGS:
+  - "debug"
+FLAGS:
+  num_train_tasks: 5
+  num_test_tasks: 10
+  strips_learner: "oracle"
+  timeout: 100
+  llm_model_name: "text-davinci-002"
+  sesame_use_visited_state_set: True  # because envs are PDDL-only
+  llm_use_cache_only: True  # because we already ran experiments once
```
Comment (ronuchit) [inline scripts/configs/llm_pddl.yaml:117]: put results in description

File: scripts/configs/llm_pddl.yaml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+      pddl_spanner_procedural_test_min_extra_spanners: 0
+      pddl_spanner_procedural_test_max_extra_spanners: 5
+      pddl_spanner_procedural_test_min_locs: 5
+      pddl_spanner_procedural_test_max_locs: 6
+ARGS:
+  - "debug"
+FLAGS:
+  num_train_tasks: 5
+  num_test_tasks: 10
+  strips_learner: "oracle"
```
Comment (ronuchit) [inline scripts/configs/llm_pddl.yaml:113]: wait...why would LLMs need this?!

File: scripts/configs/llm_pddl.yaml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+      pddl_spanner_procedural_train_min_locs: 2
+      pddl_spanner_procedural_train_max_locs: 4
+      pddl_spanner_procedural_test_min_nuts: 6
+      pddl_spanner_procedural_test_max_nuts: 7
+      pddl_spanner_procedural_test_min_extra_spanners: 0
+      pddl_spanner_procedural_test_max_extra_spanners: 5
+      pddl_spanner_procedural_test_min_locs: 5
+      pddl_spanner_procedural_test_max_locs: 6
+ARGS:
+  - "debug"
```
Comment (ronuchit) [inline scripts/configs/llm_pddl.yaml:109]: what does this do

**Review by Alchemy5 — APPROVED** [review]

File: scripts/configs/llm_pddl.yaml (hunk 3 above)
Comment (tomsilver) [reply scripts/configs/llm_pddl.yaml:109]: it's equivalent to `--debug`, which in this case is helpful for spitting out extra info during the LLM querying

File: scripts/configs/llm_pddl.yaml (hunk 2 above)
Comment (tomsilver) [reply scripts/configs/llm_pddl.yaml:113]: are you asking why the LLMs need to know operators?

Comment (tomsilver) [reply scripts/configs/llm_pddl.yaml:113]: (it's because the output of the LLM is converted into operators)

**Review by NishanthJKumar — APPROVED** [review]

**Pull request merged**